src/main/java/com/hollingsworth/arsnouveau/common/items/SpellCrossbow.java

```diff
@@ -81,14 +82,18 @@ public void releaseUsing(@NotNull ItemStack pStack, @NotNull Level pLevel, @NotN
     }
 
     private boolean tryLoadProjectiles(LivingEntity pShooter, ItemStack pCrossbowStack) {
+        if(pShooter.level().isClientSide){
```

Comment:
What is this for? Vanilla does not side check here for the bow, is this fixing some other behavior?

Reply:
if you look down a bit, you could see that I am trying to cast Level into ServerLevel. This method is called on both sides (I think for animation purposes and probably other things) so this check ensures that spell related code is run only on the server